storage/scylladb_client.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#ifndef TA_SCYLLADB_CLIENT_H_
+#define TA_SCYLLADB_CLIENT_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "scylladb_utils.h"
+
+typedef struct {
+  CassCluster* cluster;
+  CassSession* session;
+  char* host;
+  bool enabled; /**< switch of db connection */
+} db_client_service_t;
+
+/**
+ * @brief init ScyllaDB client serivce and connect to specific cluster
+ *
+ * @param[out] service ScyllaDB client service
+ * @return
+ * - SC_OK on success
+ * - non-zero on error
+ */
+status_t db_client_service_init(db_client_service_t* service);
+
+/**
+ * @brief free Scylla client serivce
```

Comment:
Should be "ScyllaDB".

Reply:
Got it. I will change all of "scylla", "Scylla" ... to "ScyllaDB"